src/figuresClasses.ts

```diff
@@ -1,19 +1,94 @@
+enum Shape {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle'
+}
+
+enum Color {
+  red = 'red',
+  green = 'green',
+  blue = 'blue'
+}
 export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
 
+function roundedDown(number: number): number {
+  return Math.floor(number * 100) / 100;
 }
 
-export class Triangle {
+function negativeNumber(numbers: number[]):boolean {
```

Comment:
```suggestion
function checkHasNegativeNumber(numbers: number[]): boolean {
```
use verbs in function names that describe what the function does
`has` because returns boolean, and you check the array, not a single number

Just FYI